client/src/components/Modal/Modal.js

```diff
@@ -49,24 +40,15 @@ const Modal = ({
     <>
       {modalOpen ? (
```

Comment:
a better convention would be to use && instead of an empty if else

```suggestion
      {modalOpen && (
```

Reply:
But then it doesn´t work..